.gitignore

```diff
@@ -1,2 +1,7 @@
 *.pyc
-**/__pycache__
\ No newline at end of file
+**/__pycache__
+internal/web.py.bak
+.idea/
+internal/static/cartdata/
+data/
```

Comment:
Please remove this from the .gitignore. These files must be tracked as a part of the repository.